netlify.toml

```diff
@@ -0,0 +1,6 @@
+[build]
+  base = "src/frontend/next"
+  publish = "src/frontend/next/public"
+  command = "npm run build"
+
+
```

Comment:
We probably need the api URL here from the env:

```
[build.environment]
  API_URL="https://dev.telescope.cdot.systems"
```

There might be others?

Reply:
I actually think that's probably it, based on the Vercel config: https://github.com/Seneca-CDOT/telescope/blob/master/now.json

Reply:
Good thinking, will add to see it all work together now (fixed deployment settings).